src/main/java/org/myrobotlab/service/ProgramAB.java

```diff
@@ -601,16 +610,16 @@ public void startSession(String path, String userName, String botName) {
     this.setCurrentBotName(botName);
     this.setCurrentUserName(userName);
 
-    if (!checkBrain(botName)) {
-      return;
-    }
+//    if (!checkBrain(botName)) {
```

Comment:
AIMLIF folder will no longer exist.  All of the code related to it should be removed.. including this check which incorrectly returns false if you specify a different than expected path

Reply:
I think it is important to check for aimlif and move/del if exist. Because this prevents normal loading.
I'm not sure everyone will think to erase it from itself. 
Often the sets are copied from one version to another. during migration from manticore to nixie it may be catastrophic ...